USERS.md

```diff
@@ -389,4 +389,5 @@ Currently, the following organizations are **officially** using Argo CD:
 1. [Yubo](https://www.yubo.live/)
 1. [ZDF](https://www.zdf.de/)
 1. [Zimpler](https://www.zimpler.com/)
+1. [ZipRecuiter](https://www.ziprecruiter.com/)
```

Comment:
Let's have a separate PR for this, so that it doesn't get reverted unintentionally if this PR has any issues.

Reply:
I've opened https://github.com/argoproj/argo-cd/pull/21076